### DISCUSSION TITLE
Hide relationships to the "outer" tables

### DISCUSSION THREAD

**Discussion by Yuri05** [schemacrawler/SchemaCrawler#818] [Q&A] [answered]
Hi, I am trying to visualize a **subset of tables** with their relationships in a sqlite database, and I want to see the relationships only between the **shown** tables

E.g. here I filter to the 2 tables : **tab_observers** and **tab_observer_rates**
`schemacrawler.bat --server=sqlite --database=DB.sqlite --info-level=standard --command=schema --portable-names --no-info=true --tables=".*tab_observers|.*tab_observer_rates" --output-file=obs.png`

The result is:

![00_obs](https://user-images.githubusercontent.com/25061876/182666292-1c3a97a5-7c11-4a6a-bd21-bfdb46f4add6.png)

I would like to hide the relationships between the tables which are not shown (**tab_calculation_method_rates** etc.).
Thus in this simple case I would like to see only this:
![grafik](https://user-images.githubusercontent.com/25061876/182666452-d6027a0d-9b8d-4767-a22a-c10f5bc59a09.png)

Is it possible?
Thank you.

**Comment by sualeh** [comment, marked as answer]
@Yuri05 try editing "config/schemacrawler.config.properties", and uncomment the following setting, and make it false. Please let me know if that works for you.

`schemacrawler.graph.show.foreignkey.filtered_tables=false`

**Reply by Yuri05** [reply]
Yes, this perfectly worked. Thank you!
![grafik](https://user-images.githubusercontent.com/25061876/182799831-b4dfa0b8-44a3-45fb-917e-d4d298e8b3d9.png)
